src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
+{
+    public const FIELD_NAME = 'SudoModePasswordField';
+    
+    protected $schemaComponent = 'SudoModePasswordField';
+
+    public function __construct($name, $title = null, $value = '')
```

Comment:
```suggestion
    public function __construct()
```
As per https://github.com/silverstripe/silverstripe-framework/pull/11582#discussion_r1957440288

Reply:
Done